packages/sdk/client/src/echo/util.ts

```diff
@@ -23,3 +23,13 @@ export const getSpace = (object: ReactiveObject<any>): Space | undefined => {
 };
 
 export const isSpace = (object: unknown): object is Space => object instanceof SpaceProxy;
+
+/**
+ * Fully qualified id of a reactive object is a combination of the space key and the object id.
+ *
+ * @returns Fully qualified id of a reactive object.
+ */
+export const fullyQualifiedId = (object: ReactiveObject<any>): string => {
+  const space = getSpace(object);
+  return space ? `${space.key.toHex()}:${object.id}` : object.id;
```

Comment:
`app-framework`’s `navigation.ts` currently defines separator constants to help parse URI’s, e.g. `SLUG_LIST_SEPARATOR`. If this will end up in a URL, I think this will need some coordination.